app/renderer/components/navigation/urlBar.js

```diff
@@ -428,6 +398,16 @@ class UrlBar extends React.Component {
     contextMenus.onUrlBarContextMenu(e)
   }
 
+  onStop () {
+    ipc.emit(messages.SHORTCUT_ACTIVE_FRAME_STOP)
```

Comment:
yea, that will have the added benefit of calling one windowAction instead of 2. You can refactor `WINDOW_SET_URL_BAR_SELECTED` and `WINDOW_SET_URL_BAR_ACTIVE` handlers in urlBarReducer into methods so you can call them from `WINDOW_ON_STOP`. In a follow-up we can get rid of `SHORTCUT_ACTIVE_FRAME_STOP` and replace it with a `WINDOW_ON_STOP` handler in the browser process